lisa/features/sriov.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any
+
+from lisa.feature import Feature
+from lisa.platform_ import Platform
+from lisa.sut_orchestrator.azure.common import get_network_client
+from lisa.util import SkippedException
+from lisa.util.logger import get_logger
+
+FEATURE_NAME_SRIOV = "Sriov"
+
+
+class Sriov(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
+        super().__init__(node, platform)
+        self._log = get_logger("feature", self.name(), self._node.log)
```

Comment:
We'd better move the line above to the base, no? Also, no volunteers to fix the Any/Any above too? I'll do that soon, if not.

Reply:
fixed in my branch, not push yet.